src/nuget/Microsoft.DotNet.BuildTools.nuspec

```diff
@@ -2,7 +2,7 @@
 <package>
   <metadata minClientVersion="2.8.1">
     <id>Microsoft.DotNet.BuildTools</id>
-    <version>1.0.25-prerelease</version>
+    <version>1.0.26-prerelease</version>
```

Comment:
It is not.  The daily build will append a build number so there is no need to bump patch number at this time.